### ISSUE TITLE
Remote Site Settting issue

### ISSUE THREAD

**Issue by Uncrasher** [haripriyamurthy/CustomMetadataLoader#4]
Hi,
I am getting the following error when I click the Create Remote Site Setting.  Also in the error dialog section the last bullet point states the following...
Press the Create Remote Site Setting button to perform this step or refer to the post install step below to perform this manually.

Not sure where to go!

ERROR after clicking he button....
Cannot create a new component with the namespace: <Mynamespace>. Only components in the same namespace as the organization can be created through the API

**Comment by haripriyamurthy** [comment]
You can create a remote site setting using these steps:
1. Go to Security Controls -> Remote Site Settings
2. Create a new remote site setting with the 'remote site url' being the url part before "/apex/custommetadatarecorduploader" when you load the Custom Metadata Loader Tab. It will look something like "https://NAMESPACE.INSTANCE.visual.force.com".

Can you let me know what is special about your org where you are seeing this error? Did you package this data loader? Is the data loader's namespace different from the org's namespace?

**Comment by Uncrasher** [comment]
That did the trick - thank you for your help.

**Comment by Nikhil-93** [comment]
Hi @haripriyamurthy : 

I am also getting following error:
Error:
Cannot create a new component with the namespace: wubs. Only components in the same namespace as the organization can be created through the API.
I have deployed using workbench and I created remote site setting also as per your post with the 'remote site url' same as of my sandbox domain url.
Could you please help me if I am missing anything here. Thanks.

**Comment by haripriyamurthy** [comment]
@Nikhil-93 When do you see this error? Creating a remote site setting manually should solve the problem above.

**Comment by Nikhil-93** [comment]
@haripriyamurthy : It's fixed now. Actually issue was that there were some letters in uppercase in domain url and browser automatically converts uppercase letters into lowercase. For eg. browse was redirecting at "https://abcdev.cs40.my.salesforce.com" and when I checked for actual endpoint on workbench, it was showing "https://ABCDev.cs40.my.salesforce.com". Once I updated actual endpoint, it worked good for me. 
Thanks.
